### PR TITLE
[codegen/nodejs] - Fix typescript enum issues

### PR DESCRIPTION
**Pull request by komalali**
Fixes: https://github.com/pulumi/pulumi/issues/5695

### CODE REVIEW

File: pkg/codegen/nodejs/gen.go [hunk 1]
```diff
@@ -995,12 +1001,12 @@ func (mod *modContext) sdkImports(nested, utilities bool) []string {
...
 	if nested {
-		imports = append(imports, fmt.Sprintf("import * as inputs from \"%s/types/input\";", relRoot))
-		imports = append(imports, fmt.Sprintf("import * as outputs from \"%s/types/output\";", relRoot))
+		enumsImport := ""
 		containsEnums := mod.pkg.Language["nodejs"].(NodePackageInfo).ContainsEnums
 		if containsEnums {
-			imports = append(imports, fmt.Sprintf("import * as enums from \"%s/types/enums\";", relRoot))
+			enumsImport = ", enums"
 		}
+		imports = append(imports, fmt.Sprintf("import { input as inputs, output as outputs%s } from \"%s/types\";", enumsImport, relRoot))
```
Comment (lblackstone) [inline pkg/codegen/nodejs/gen.go:1009]: nit: If you use a raw string, you don't have to escape the quotes.
```suggestion
		imports = append(imports, fmt.Sprintf(`import { input as inputs, output as outputs%s } from "%s/types";`, enumsImport, relRoot))
```

File: sdk/go/pulumi/types_builtins.go [hunk 2]
```diff
@@ -2591,7 +2591,7 @@ type BoolInput interface {
 type Bool bool
 
 var (
-	True = BoolPtr(true)
+	True  = BoolPtr(true)
```
Comment (komalali) [inline sdk/go/pulumi/types_builtins.go:2594]: The lint step was breaking because this file wasn't gofmt'd

File: pkg/codegen/nodejs/gen.go [hunk 3]
```diff
@@ -85,6 +85,10 @@ type modContext struct {
 	disableUnionOutputTypes bool              // Disable unions in output types.
 }
 
+func (mod *modContext) String() string {
```
Comment (komalali) [inline pkg/codegen/nodejs/gen.go:88]: This makes debugging easier by putting the mod names next to the type in the debugger window.

**Review by lblackstone — APPROVED** [review]

**Comment by mikhailshilkov** [comment]
The change looks good. My only concern is that pretty much every TS file is going to change, which leads to huge diffs once. Was the formatting change purely stylistic or was something wrong with the old way (apart from missing the `enum` import)?

**Comment by komalali** [comment]
@mikhailshilkov yes the other changes are stylistic, but they improve readability and follow best practices. Changes like this are not uncommon, I think we can stomach it.

**Pull request merged**